build.gradle

```diff
@@ -175,8 +175,13 @@ subprojects {
             cacheChangingModulesFor 5, 'days'
         }
 
-        exclude(group: "ch.qos.logback", module: "logback-core")
-        exclude(group: "ch.qos.logback", module: "logback-classic")
+        if (!project.name.contains('cassandra-ticket-registry')) {
```

Comment:
Your solution is perfectly found as is. It is however not sustainable because it promotes "a bad habit". This is the principal of information hiding where each module should only be concerned about its own stuff and not leak outside. I understand it's a lot of work and I am happy to help where I can, but I do think this is the right way to go about this and keep sanity in the build. I understand it might seem like a small thing right now and insignificant. Small things pile up over time and this is mostly foreseen pain.